test/client/fixtures/runner/automation/regression-test.js

```diff
@@ -262,10 +262,9 @@ $(document).ready(function () {
                 .addClass(TEST_ELEMENT_CLASS)
                 .click(function () {
                     clicked = true;
-                })
-                .appendTo('body');
+                });
 
-        window.setTimeout(function () {
+        $iframe[0].onload = function () {
```

Comment:
Please use the [waitForIframe](https://github.com/AlexanderMoskovkin/qunit-harness#wait-for-an-iframe-action) function. It's availabe in `qunit-harness` yet.